deploy/helm/kubecf/assets/operations/instance_groups/smoke-tests.yaml

```diff
@@ -2,6 +2,10 @@
   path: /instance_groups/name=smoke-tests/jobs/name=cf-cli-6-linux/properties?/quarks/bpm/processes
   value: []
 
+- type: replace
+  path: /instance_groups/name=smoke-tests/env?/bosh/agent/settings/disable_log_sidecar
```

Comment:
:heavy_plus_sign:

Reply:
Will we need to update CI to use another mechanism to determine if the test succeeded or failed given the pod is deleted without the logs container?